docs/source/pickling_cuml_models.ipynb

```diff
@@ -0,0 +1,209 @@
+{
```

Comment:
<p>I think you missed a word in the final sentence here:</p><p>"can be used to <strong>perform</strong> distributed inference on a Dask cluster"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cuml/pull/3154/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>